cmake/defaults/Packages.cmake

```diff
@@ -251,7 +258,22 @@ if (PXR_BUILD_IMAGING)
     endif()
     # --Opensubdiv
     set(OPENSUBDIV_USE_GPU ${PXR_BUILD_GPU_SUPPORT})
-    find_package(OpenSubdiv 3 REQUIRED)
+    find_package(OpenSubdiv 3 CONFIG)
+    if(OpenSubdiv_DIR)
+        set(OPENSUBDIV_LIBRARIES OpenSubdiv::osdCPU OpenSubdiv::osdGPU)
+        set(OPENSUBDIV_OSDCPU_LIBRARY OpenSubdiv::osdCPU)
+        message(STATUS "OpenSubdiv found as config, OpenSubdiv_DIR is ${OpenSubdiv_DIR}")
+        foreach(t OpenSubdiv::osdCPU OpenSubdiv::osdGPU OpenSubdiv::osdCPU_static OpenSubdiv::osdGPU_static OpenSubdiv::OpenSubdiv)
+            if(TARGET "${t}")
+                message(STATUS "${t} is TARGET")
+            else()
+                message(STATUS "${t} is not TARGET")
+            endif()
+        endforeach()
+    else()
+        find_package(OpenSubdiv 3 REQUIRED)
+        message(STATUS "OpenSubdiv found as module")
```

Comment:
Ditto above - can we lower to `VERBOSE` or `DEBUG`?

Reply:
also debugging, will be removed

Reply:
removed